src/index.html

```diff
@@ -1,13 +1,585 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dia</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+    <title>Air</title>
+    <link rel="icon" href="images/logo/air-logo.svg">
+    <link rel="stylesheet" href="./styles/main.scss" >
+    <script
+			type="text/javascript"
+			src="scripts/main.js"
+			defer
+		></script>
```

Comment:
Need to fix the indents here
```suggestion
    <script
      type="text/javascript"
      src="scripts/main.js"
      defer
    ></script>
```